releases/release-4.0.16.md

```diff
@@ -0,0 +1,97 @@
+        - Fix a bug that TiCDC could meet replication interruption when multiple TiKVs crash or forcing restart. [#3290](https://github.com/pingcap/ticdc/pull/3290)
+        - Fix memory leak after processing DDLs. [#3274](https://github.com/pingcap/ticdc/pull/3274)
+        - Please add a release note. If you don't think this PR needs a release note then fill it with `None`. [#3267](https://github.com/pingcap/ticdc/pull/3267)
+        - [#3210](https://github.com/pingcap/ticdc/pull/3210)
+        - Please add a release note. If you don't think this PR needs a release note then fill it with `None`. [#3176](https://github.com/pingcap/ticdc/pull/3176)
+        - bugfix: fix changefeed does not fast fail when occur ErrGCTTLExceeded error. [#3134](https://github.com/pingcap/ticdc/pull/3134)
+        - Optimize the rate limit control when TiKV reloads and fix the congestion in gPRC, which may cause slow initialization phase. [#3131](https://github.com/pingcap/ticdc/pull/3131)
+        - Change Kafka sink default `MaxMessageBytes` to 1MB. [#3106](https://github.com/pingcap/ticdc/pull/3106)
+        - Fix the bug that fallback resolvedTs event  will block the progress of resolve lock when occur region merging [#3099](https://github.com/pingcap/ticdc/pull/3099)
+        - Close gPRC stream and re-create it when meeting `ErrPrewriteNotMatch` to avoid duplicated request error [#3094](https://github.com/pingcap/ticdc/pull/3094)
+        - Fix kafka sink can not send message due to constraint by `max-message-size` option. [#3046](https://github.com/pingcap/ticdc/pull/3046)
+        - Add metrics to observe incremental scan remaining time [#3032](https://github.com/pingcap/ticdc/pull/3032)
+        - Fix tikv_cdc_min_resolved_ts_no_change_for_1m keep firing when there is no changefeed. [#3023](https://github.com/pingcap/ticdc/pull/3023)
+        - Fix possible deadlocking when Kafka producer reports an error. [#3015](https://github.com/pingcap/ticdc/pull/3015)
+        - Release new owner and processor implementation to release-5.0.
+        - Highly available model and core modules refactoring.[#1927](https://github.com/pingcap/ticdc/pull/1927) [#3014](https://github.com/pingcap/ticdc/pull/3014)
+        - Ignore the global flag for changefeed update command. [#2875](https://github.com/pingcap/ticdc/pull/2875)
+        - Fix dml is not replicated after adding partition in partition table without valid index [#2863](https://github.com/pingcap/ticdc/pull/2863)
+        - Extend creating service gc safepoint ttl to 1 hr to support creating changefeeds that needs long initialization time. [#2851](https://github.com/pingcap/ticdc/pull/2851)
+        - Fix memory leak which may happen in create new changefeed. [#2623](https://github.com/pingcap/ticdc/pull/2623)
+        - Prohibit operating TiCDC clusters across major and minor versions [#2601](https://github.com/pingcap/ticdc/pull/2601)
```

Comment:
```suggestion
## Improvements
+ Tools

    + BR

        - Increase the robustness for restoring. [#1445](https://github.com/pingcap/br/pull/1445)
  
    + TiCDC
        - Add rate limiter to limit EtcdWorker tick frequency. [#3267](https://github.com/pingcap/ticdc/pull/3267)
        - Optimize the rate limit control when TiKV reloads and fix the congestion in gPRC, which may cause slow initialization phase. [#3131](https://github.com/pingcap/ticdc/pull/3131)
        - Change Kafka sink default `MaxMessageBytes` to 1MB. [#3106](https://github.com/pingcap/ticdc/pull/3106)
        - Ignore the global flag for changefeed update command. [#2875](https://github.com/pingcap/ticdc/pull/2875)
        - Extend creating service gc safepoint ttl to 1 hr to support creating changefeeds that needs long initialization time. [#2851](https://github.com/pingcap/ticdc/pull/2851)
        - Prohibit operating TiCDC clusters across major and minor versions. [#2601](https://github.com/pingcap/ticdc/pull/2601)

## Bug Fixes
+ Tools

    + TiCDC
        - Fix changefeed checkpoint lag negative value error. [#3532](https://github.com/pingcap/ticdc/pull/3532) 
        - Fix OOM in container environments. [#3440](https://github.com/pingcap/ticdc/pull/3440)
        - Fix TiCDC could meet replication interruption when multiple TiKVs crash or forcing restart. [#3290](https://github.com/pingcap/ticdc/pull/3290)  
        - Fix memory leak after processing DDLs. [#3274](https://github.com/pingcap/ticdc/pull/3274)
        - Fix changefeed does not fast fail when occur ErrGCTTLExceeded error. [#3134](https://github.com/pingcap/ticdc/pull/3134)  
        - Fix fallback resolvedTs event will block the progress of resolve lock when occur region merging. [#3099](https://github.com/pingcap/ticdc/pull/3099)
        - Close gPRC stream and re-create it when meeting `ErrPrewriteNotMatch` to avoid duplicated request error [#3094](https://github.com/pingcap/ticdc/pull/3094)
        - Fix kafka sink can not send message due to constraint by `max-message-size` option. [#3046](https://github.com/pingcap/ticdc/pull/3046)
        - Fix tikv_cdc_min_resolved_ts_no_change_for_1m keep firing when there is no changefeed. [#3023](https://github.com/pingcap/ticdc/pull/3023)
        - Fix possible deadlocking when Kafka producer reports an error. [#3015](https://github.com/pingcap/ticdc/pull/3015)
        - Fix dml is not replicated after adding partition in partition table without valid index. [#2863](https://github.com/pingcap/ticdc/pull/2863)
        - Fix memory leak which may happen in create new changefeed. [#2623](https://github.com/pingcap/ticdc/pull/2623)
```